manifests/minimal-master-replica-svcmonitor.yaml

```diff
@@ -0,0 +1,145 @@
+# Here we use https://github.com/prometheus-community/helm-charts/charts/kube-prometheus-stack
+# helm install kps . -n kps --create-namespace
```

Comment:
what's kps? :smiley:
Better use `postgres-operator` or so.

Reply:
The line is an example of installing `kube-prometheus-stack`, I removed it due to this can make confusion, and removed the useless labels.